### PR TITLE
[SIG-1926] now using ASC Heading for melding form title

### PR DESCRIPTION
**Pull request by jasperswart**
- added ASC Heading to IncidentWizard
- added data-testid to IncidentForm and  IncidentPreview
- updated  snapshot of IncidentForm and  IncidentPreview
- removed snapshot test from IncidentWizard

PLEASE NOTE:
can only be merged when #554 is merged
it needs data-testid="incidentForm" in the incidentForm component
this branch will keep failing at snapshots in incidentForm until #554 is merged

### CODE REVIEW

File: src/signals/incident/components/IncidentWizard/index.js [hunk 1]
```diff
@@ -1,21 +1,27 @@
...
 
 import LoadingIndicator from 'shared/components/LoadingIndicator';
 
 import IncidentForm from '../IncidentForm';
 import IncidentPreview from '../IncidentPreview';
 import onNext from './services/on-next';
 
-import './style.scss';
+const StyledH1 = styled(Heading)`
+  margin-top: ${themeSpacing(6)};;
```
Comment (janjaap) [inline src/signals/incident/components/IncidentWizard/index.js]: One semicolon too many

File: src/signals/incident/components/IncidentWizard/index.js [hunk 2]
```diff
@@ -1,21 +1,27 @@
...
+  margin-bottom: 30px;
+
+  @media screen and (max-width: 540px) {
```
Comment (janjaap) [inline src/signals/incident/components/IncidentWizard/index.js]: There is no need to need to define a media query here; the `Heading` component already defines the font-size for specific breakpoints in the theme configuration.

Comment (jasperswart) [reply src/signals/incident/components/IncidentWizard/index.js]: I thought that it wasn't working ub I see now it is

File: src/signals/incident/components/IncidentWizard/index.js (hunk 1 above)
Comment (jasperswart) [reply src/signals/incident/components/IncidentWizard/index.js]: gedaan

File: src/signals/incident/components/IncidentWizard/index.js (hunk 2 above)
Comment (jasperswart) [reply src/signals/incident/components/IncidentWizard/index.js]: gedaan

File: src/signals/incident/components/IncidentWizard/index.js [hunk 3]
```diff
@@ -1,21 +1,20 @@
...
 import LoadingIndicator from 'shared/components/LoadingIndicator';
 
 import IncidentForm from '../IncidentForm';
 import IncidentPreview from '../IncidentPreview';
 import onNext from './services/on-next';
 
-import './style.scss';
+const StyledH1 = styled(Heading)`
+  margin-top: ${themeSpacing(6)};
+  margin-bottom: 30px;
```
Comment (janjaap) [inline src/signals/incident/components/IncidentWizard/index.js]: `themeSpacing` Should be used here
🤷‍♂

**Comment by janjaap** [comment]
Build is breaking for this PR

File: src/signals/incident/components/IncidentPreview/index.test.js [hunk 4]
```diff
@@ -44,45 +50,43 @@ describe('<IncidentPreview />', () => {
...
     it('expect to render correctly', () => {
       isVisible.mockImplementation(() => true);
-      const wrapper = shallow(
-        <IncidentPreview {...props} />
+      const { queryByText } = render(
+        withAppContext(<IncidentPreview {...props} />)
       );
 
-      expect(wrapper).toMatchSnapshot();
+      expect(queryByText('Uw (mobiele) telefoon')).toBeInTheDocument();
```
Comment (janjaap) [inline src/signals/incident/components/IncidentPreview/index.test.js]: Can you reference the values that are in the `props` var? At least rules out ambiguity.

File: src/signals/incident/components/IncidentPreview/index.test.js [hunk 5]
```diff
@@ -44,45 +50,43 @@ describe('<IncidentPreview />', () => {
...
-      },
-    });
-
-    withWizard.find('button').simulate('click');
+    fireEvent.click(
+      container.querySelectorAll('button')[1]
+    );
 
-    expect(historySpy.push).toHaveBeenCalledWith('/incident/step2');
+    expect(push).toHaveBeenCalledWith('/incident/step2');
```
Comment (janjaap) [inline src/signals/incident/components/IncidentPreview/index.test.js]: Can you verify that push hasn't been called before the click event was dispatched?

File: src/signals/incident/components/IncidentPreview/index.js [hunk 6]
```diff
@@ -1,43 +1,33 @@
...
-
-import { WithWizard } from 'react-albus';
+import { useHistory } from 'react-router-dom';
 
 import isVisible from './services/is-visible';
 
 import './style.scss';
 
 function IncidentPreview({ incidentContainer, preview, isAuthenticated }) {
+  const history = useHistory();
```
[the remark this answers is not in the document]
Comment (jasperswart) [reply src/signals/incident/components/IncidentPreview/index.js]: getting rid of  WithWizard as much as possible: it is cleaner like this and it is still working as intended

File: src/signals/incident/components/IncidentPreview/index.test.js (hunk 4 above)
Comment (jasperswart) [reply src/signals/incident/components/IncidentPreview/index.test.js]: gedaan

File: src/signals/incident/components/IncidentPreview/index.test.js (hunk 5 above)
Comment (jasperswart) [reply src/signals/incident/components/IncidentPreview/index.test.js]: gedaan

File: src/signals/incident/components/IncidentPreview/index.test.js (hunk 4 above)
Comment (janjaap) [reply src/signals/incident/components/IncidentPreview/index.test.js]: I don't see any changes. Forgot to push?

File: src/signals/incident/components/IncidentWizard/index.js (hunk 3 above)
Comment (janjaap) [reply src/signals/incident/components/IncidentWizard/index.js]: I don't see any changes. Forgot to push?

**Pull request merged**